superset-frontend/src/components/ReportModal/HeaderReportActionsDropdown/index.tsx

```diff
@@ -60,14 +60,15 @@ export default function HeaderReportActionsDropDown({
   };
 
   const menu = () => (
-    <Menu selectable={false}>
+    <Menu selectable={false} css={{ width: '200px' }}>
       <Menu.Item>
         {t('Email reports active')}
         <Switch
           data-test="toggle-active"
           checked={report?.active}
           onClick={(checked: boolean) => toggleActiveKey(report, checked)}
           size="small"
+          css={{ marginLeft: '8px' }}
```

Comment:
Same here?

Also, should we use grid units here?

Reply:
+1 on the grid units

Reply:
good point! Added, ty.